tests/qnodes/test_qnode_base.py

```diff
@@ -348,24 +348,6 @@ def circuit(x):
 class TestQNodeExceptions:
     """Tests that QNode raises proper errors"""
 
-    def test_current_context_modified_outside_construct(
-        self, operable_mock_device_2_wires, monkeypatch
-    ):
-        """Error: _current_context was modified outside of construct."""
```

Comment:
Is this edge case no longer possible?

Reply:
I just assumed that nobody messes with `QueuingContext._active_contexts`. If they do it's their fault.